plat/renesas/rcar/rcar_private.h

```diff
@@ -158,6 +158,14 @@ uint32_t rcar_get_spsr_for_bl33_entry(void);
 /* Declarations for rcar_scpi.c */
 uint32_t rcar_trigger_scpi(uint32_t x1, uint32_t x2, uint32_t x3, uint32_t x4);
 
+/* Declarations for rcar_clocks.c */
+int rcar_clock_nr_clocks(void);
+uint32_t rcar_clock_get_min_rate(int clocknr);
```

Comment:
`int clocknr` implies that you can have `clocknr < 0`?

Reply:
No, clocknr certainly can't be a negative value. Will recheck.